New log file opened at Mon Feb 22 18:52:12 CST 2016 by 6305@sharukh-MacBookPro
Taking next connection to: 48072
New client attempting to connect on port: 48442-48072
  Address = /127.0.1.1:48072
This client on port 48442 has uuid 35989a5b-0616-4271-bb43-f2eb6556e9ab configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-35989a5b-0616-4271-bb43-f2eb6556e9ab
Taking next connection to: 48072
New client attempting to connect on port: 48621-48072
  Address = /127.0.1.1:48072
Handshake exception on socket: 48621
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 48072
New client attempting to connect on port: 48622-48072
  Address = /127.0.1.1:48072
This client on port 48622 has uuid c4c00212-7755-426a-9c23-413a9f88ddf2 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-35989a5b-0616-4271-bb43-f2eb6556e9ab, play-fork-run-c4c00212-7755-426a-9c23-413a9f88ddf2
Taking next connection to: 48072
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 48072
